types/amplifier/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "ban-types": false,
```

Comment:
Is there a reason why disabling this rules?

Reply:
No idea, i copied the file from @types/amplify